### PR TITLE
Create a producer and associate it as a group supplier

### PR DESCRIPTION
**Pull request by enricostano**

### CODE REVIEW

File: src/containers/Group/Providers/Base.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+      if (Object.keys(errors).length) {
+        return Promise.reject(errors);
+      }
+
+      // do something on success
+      // self.props.initialize('onboardingCreateGroup', {});
+
+      return Promise.resolve({});
+    }).then(() => {
```
Comment (andresgutgon) [inline src/containers/Group/Providers/Base.js]: Que pasa si pones aqui?

``` javascript
}).then((createdProducerData) => {
```

Comment (enricostano) [reply src/containers/Group/Providers/Base.js]: Acabo de probarlo, y esta' vacio

File: src/containers/Group/Providers/Base.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      const errors = self.props.createProducerErrors;
+
+      if (Object.keys(errors).length) {
+        return Promise.reject(errors);
+      }
+
+      // do something on success
+      // self.props.initialize('onboardingCreateGroup', {});
+
+      return Promise.resolve({});
```
Comment (andresgutgon) [inline src/containers/Group/Providers/Base.js]: What happen if you put the `data` here?

Comment (enricostano) [reply src/containers/Group/Providers/Base.js]: But `data` is the initial data that we send in the first request.

Comment (enricostano) [reply src/containers/Group/Providers/Base.js]: Actually if I put `data` there, I get is as an argument in the following promise... but still, it's just the initial data.

File: src/containers/Group/Providers/Base.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    createProducerErrors: PropTypes.object,
+    createSupplierErrors: PropTypes.object,
+  }
+
+  handleSubmit(data) {
+    const self = this;
+
+    data.group_id = this.props.group.id;
+
+    return this.props.createProducer(data).then(() => {
```
Comment (andresgutgon) [inline src/containers/Group/Providers/Base.js]: What happen if you put `data` in this `then`? Do you see the pattern? :trollface:

Comment (enricostano) [reply src/containers/Group/Providers/Base.js]: got it, thanks! :heart_eyes:

Comment (andresgutgon) [reply src/containers/Group/Providers/Base.js]: LOL

**Pull request closed without merging**